styles/styles-grid.css

```diff
@@ -0,0 +1,426 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+
+:root {
+  --text-color: #383838;
+  --general-white: #ffffff;
+}
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  height: 100vh;
+}
+
+.svg-sprites {
+  position: absolute;
+  visibility: hidden;
+  width: 0;
+  height: 0;
+}
+
+.main {
+  background-image: url(../image/header/background-header.png);
+  background-repeat: no-repeat;
+}
```

Comment:
The background image is not centred nor stretched or repeated. So, it looks not good on wide screens.